Library/Homebrew/cmd/deps.rb

```diff
@@ -200,24 +209,40 @@ def puts_deps(dependents, args:, recursive: false)
     end
   end
 
+  def open_deps_digraph(dependents, recursive:)
+    dependency_graph = Utils::TopologicalHash.graph_package_dependencies(dependents)
+    dependency_graph.select! { |d, _| dependents.include?(d) } unless recursive
+
+    graphviz_code = dependency_graph.map do |d, deps|
```

Comment:
I wonder if it's worth adding a way to just print the output of `graphviz_code`, since there's probably a lot more you can do with this than just pass it to `dot`.